.changeset/fair-buttons-punch.md

```diff
@@ -0,0 +1,24 @@
+---
+"@rhds/elements": minor
+---
+
+`<rh-card>`: ✨ Added Inline Promo variant
```

Comment:
@markcaron made an oopsie on review

This comment is a blocker to stop accidental auto merge. Resolve after full review has been done.

Reply:
marionne's request for changes should block merge